uwblueprint/focus-on-nature
# feat: waitlisted camper status endpoint

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-S22-1a3e651ab8544192bf0fdcbba04f6d03?p=667289acc6924a168165309cc6119507)



## Implementation description
* Updated the WaitListedCamper Model to store a status string enum that is one of `["NOT REGISTERED", "REGISTRATION FORM SENT", "REGISTERED"]` with a default value of `"NOT REGISTERED"`



## Steps to test
1. Grab the ID of a waitlisted camper from MongoDB or use the following one: `623baa83382f42b34e00d5c7`
2. Send a put request to the following url: `http://localhost:5000/campers/waitlist/:id`
3. Return of the endpoint should be the updated waitlisted camper with the status field changed to `"REGISTRATION FORM SENT"`



## What should reviewers focus on?
* How cool this is :)


## Checklist
